LondonBritishLibrary/orient/BLorient8822.xml

```diff
@@ -0,0 +1,162 @@
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+                            Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                                licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <msDesc xml:id="ms">
+                    <msIdentifier>
+                        <repository ref="INS00001BL"/>
+                        <collection>Oriental</collection>
+                        <idno>BL Oriental 8822</idno>
+                    </msIdentifier>
+                    <msContents>
+                        <msItem xml:id="ms_i1">
+                            <locus from="2ra" to="35rb"/>
+                            <title ref="LIT1340EnochE"/>                  
+                        </msItem>
+                        <msItem xml:id="ms_i2">
+                            <locus from="35va" to="51va"/>
+                            <title ref="LIT1688Job"/>
+                        </msItem>
+                        <msItem xml:id="ms_i3">
+                            <locus from="51vb" to="63ra"/>
+                            <title ref="LIT3529Daniel"/>
+                        </msItem>
+                        <msItem xml:id="ms_i4">
+                            <locus from="63rb" to="79ra"/>
+                            <title ref="LIT1376Apocal"/>
+                            <incipit xml:lang="gez">አመ፡ ሰላሳ፡ አመት፡ እምዘወድቀ፡ ሀገሪትነ፡ ወሀለውኩ፡ ውስተ፡ ባቢሎን፡ አነ፡ ሰቱኤል፡ </incipit>
+                        </msItem>
+                        <msItem xml:id="ms_i5">
+                            <locus from="63rb" to="79ra"/>
+                            <title ref="LIT2358Sirach" type="incomplete">Ecclesiasticus I, I-XXI</title>
+                            <note>The spaces for rubrication have been left blank. The text ends in medio.</note>
+                            <incipit xml:lang="gez"><supplied reason="omitted" resp="Strelcyn">ቀዳሚሃ። ኵላ፡ ጥ</supplied>በብ፡ ዘተከሥተ፡ በኢይሩሳሌም፡
+                                ለእግዚአብሔር፡ ውእቱ፡ </incipit>
+                            <explicit xml:lang="gez">ዘየሐንጽ፡ ቤቶ፡ በንዋየ፡ ልቃሕ፡ ከመ፡ ንድቀ፡ ክረምት፡ </explicit>
+                        </msItem>
+                    </msContents>
+                    <physDesc>
+                        <objectDesc form="Codex">
+                            <supportDesc>
+                                <support>
+                                    <material key="parchment"/>
+                                    <material key="paper">Flyleaves of paper added in rebinding in the <placeName ref="INS00001BL"/>.</material>
```

Comment:
I think the information on the flyleaves would better be moved to the description of the binding